dayrepo/snippets/static/js/process_add.js

```diff
@@ -1,9 +1,28 @@
-let i = 1 ;
+const limit = 6;
+const bool_select = 8;
+const not_select = 10;
 const addForm = () => {
-  let content_area = document.getElementById("form_area");
+  const count = Number(document.getElementById("count").innerHTML);
+  let now_count = count;
+  if (now_count == limit){
+    alert("これ以上増やせません")
+    return
+  }
+  alert("工程を追加します");
+  now_count += 1;
```

Comment:
こうすると const にできて、変数も一つ減らせます〜
変数名は適当なので、好きに修正してください。

もし以下をそのまま使う場合、こちらも一緒にマージしてください: 

```suggestion
  const now_count = Number(document.getElementById("count").innerHTML);
  if (now_count == limit){
    alert("これ以上増やせません")
    return
  }
  alert("工程を追加します");
  const count_plus1 = now_count += 1;
```

Reply:
実装しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/54/commits/12d6a8f19c912cb1c754ceb1188e060aecf0f100

Reply:
動作確認しましたか？
now_count が宣言されずいきなり出てきているように見えます

Reply:
対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/54/commits/cf039fa481f1b5122f41c1d1896182caba01e84b

Reply:
count += 1; とするのではなく、
```
document.getElementById("count").innerHTML = count + 1;
  clone_element.id = count + 1;
```

とすれば大元の変数を const にできる & count += 1;  とする箇所と実際に使用する箇所が離れないと思います〜

Reply:
対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/54/commits/b7c8ba224ac1053865a24b1dc32924532f04ccc3